database/seeders/AdminSeeder.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Database\Seeders;
+
+use Illuminate\Database\Seeder;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Str;
+use Interns2025b\Models\User;
+
+class AdminSeeder extends Seeder
+{
+    public function run(): void
+    {
+        $admin = User::firstOrCreate(
+            ["email" => "admin@example.com"],
+            [
+                "first_name" => "Admin",
+                "last_name" => "Admin_Last_Name",
+                "email_verified_at" => now(),
+                "password" => Hash::make("password"),
+                "remember_token" => Str::random(10),
+            ],
+        );
+        $admin->assignRole("administrator");
+
+        $super_admin = User::firstOrCreate(
```

Comment:
Please use **camelCase**
```suggestion
        $superAdmin = User::firstOrCreate(
```